### PR TITLE
vc: move the regeneration logic to CI

### PR DESCRIPTION
**Pull request by spaceface777**

### CODE REVIEW

File: .github/workflows/gen_vc.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - uses: actions/checkout@v2
+    - name: Build V
+      run: make
+    - name: Regenerate v.c and v_win.c
+      run: |
+        COMMIT_HASH=$(git rev-parse --short HEAD)
+        COMMIT_MSG=$(git log -1 --oneline --pretty='%s' HEAD)
+
+        rm -rf vc
+        git clone https://github.com/vlang/vc
```
Comment (spytheman) [inline .github/workflows/gen_vc.yml:21]: it would be slightly faster, if you add `--depth=1` here

Comment (spytheman) [reply .github/workflows/gen_vc.yml:21]: not sure if it will allow pushing after that though 🤔

Comment (spytheman) [reply .github/workflows/gen_vc.yml:21]: in any case, we can test later

**Comment by spytheman** [comment]
Good work!

File: .github/workflows/gen_vc.yml (hunk 1 above)
Comment (spaceface777) [reply .github/workflows/gen_vc.yml:21]: I don't think you can push to a repo after a shallow clone, but I may be wrong.

**Pull request merged**